Makefile

```diff
@@ -51,7 +51,7 @@ GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
 
 # We don't want find to scan inside a bunch of directories, to accelerate the
 # 'make: Entering directory '/src/loki' phase.
-DONT_FIND := -name tools -prune -o -name vendor -prune -o -name .git -prune -o -name .cache -prune -o -name .pkg -prune -o
+DONT_FIND := -name logback -prune -o -name tools -prune -o -name vendor -prune -o -name .git -prune -o -name .cache -prune -o -name .pkg -prune -o
```

Comment:
@slim-bean I have ignored `logback` folder here so that `protoc` works fine for other `*.proto` files. 
Should we include `logback's` `*.proto` file as well in the same `make` command or should have separate `make` command for `logback`?

WDYT?